python/pyspark/sql/tests/test_unified_udf.py

```diff
@@ -351,39 +351,41 @@ def test_regular_python_udf(self):
         import pandas as pd
         import pyarrow as pa
 
-        @udf(returnType=LongType())
-        def f1(x):
-            return x + 1
+        with self.sql_conf({"spark.sql.execution.pythonUDF.arrow.enabled": False}):
```

Comment:
why we need to explicitly set it false here?
does it means some behavior change?